oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/source/ServiceInstanceRelation.java

```diff
@@ -103,6 +103,9 @@ public String getEntityId() {
     @Getter
     @Setter
     private DetectPoint detectPoint;
+    @Getter
+    @Setter
+    private String tlsMode;
```

Comment:
This and service relation source have been changed, `scope-definitions.md` doc should be updated. 

> service_relation_mtls_cpm = from(ServiceRelation.*).filter(tlsMode == "mTLS").cpm();

This example should belong to the doc I pointed to you.